drivers/gles3/shaders/screen_space_reflection.glsl

```diff
@@ -62,6 +63,12 @@ void main() {
 
 	float roughness = normal_roughness.w;
 
+	if (roughness > max_roughness) {
+		// Do not compute SSR for rough materials to improve performance at the cost of
+		// subtle artifacting.
+		return;
+	}
+
```

Comment:
```suggestion
	if (roughness > max_roughness) {
		// Do not compute SSR for rough materials to improve performance at the cost of
		// subtle artifacting.
		frag_color = vec4(0.0);
		return;
	}

```

Without writing to frag_color an early return can result in uninitialized memory leaking out.